eth/fetcher/tx_fetcher.go

```diff
@@ -281,7 +281,7 @@ func (f *TxFetcher) Enqueue(peer string, txs []*types.Transaction, direct bool)
 			// Track the transaction hash if the price is too low for us.
 			// Avoid re-request this transaction when we receive another
 			// announcement.
-			if err == core.ErrUnderpriced || err == core.ErrReplaceUnderpriced {
+			if err == txpool.ErrUnderpriced || err == txpool.ErrReplaceUnderpriced {
```

Comment:
We should move these error definitions into a common package shared by all txpool implementations.

Reply:
e.g. still in the core package.